### PR TITLE
feat: use shape targetClass instead of memberType in config

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java [hunk 1]
```diff
@@ -45,8 +46,16 @@ public EventStreamResponse fromModel(Model model) {
...
 		final Model shacl = getShaclFromModel(model);
+		final String memberType = shacl.listStatements(null, shacl.createProperty(targetClass.getURI()), (RDFNode) null)
+				.nextOptional()
+				.map(Statement::getObject)
+				.map(RDFNode::toString)
+				.orElseThrow(() -> {
+					throw new IllegalArgumentException(
+							"Could not find a targetClass in provided shape. Missing statement: "
+									+ targetClass.getURI());
+				});
```
Comment (WLefever-Cegeka) [inline ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java:58]: Purely for readability I would suggest extracting these lines in a method

File: ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java [hunk 2]
```diff
@@ -45,8 +46,16 @@ public EventStreamResponse fromModel(Model model) {
 		final String timestampPath = getResource(model, LDES_TIMESTAMP_PATH);
 		final String versionOfPath = getResource(model, LDES_VERSION_OF);
 		final List<ViewSpecification> views = getViews(model, collection);
-		final String memberType = getResource(model, MEMBER_TYPE);
 		final Model shacl = getShaclFromModel(model);
+		final String memberType = shacl.listStatements(null, shacl.createProperty(targetClass.getURI()), (RDFNode) null)
```
Comment (WLefever-Cegeka) [inline ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java:50]: If you use `shacl.listObjectsOfProperty(createProperty(targetClass.getURI()))`, you don't need the `null`

**Review by WLefever-Cegeka — APPROVED** [review]
Very nice, @Yalz !
Left some suggestions :-)

File: ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java (hunk 2 above)
Comment (Yalz) [reply ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverterImpl.java:50]: Amazing! thanks!

**Pull request merged**